src/node_process.cc

```diff
@@ -61,11 +65,11 @@ Mutex environ_mutex;
 #define CHDIR_BUFSIZE (PATH_MAX)
 #endif
 
-void Abort(const FunctionCallbackInfo<Value>& args) {
+static void Abort(const FunctionCallbackInfo<Value>& args) {
```

Comment:
Maybe `namespace { … }` would be easier than adding `static` everywhere?

Reply:
@joyeecheung Anonymous namespaces pretty much say the same thing to me – but, yeah, this is not important for me :)